werkit/aws_lambda/build.py

```diff
@@ -46,7 +47,7 @@ def create_venv_with_dependencies(
             "-m",
             "pip",
             "install",
-            f"werkit=={importlib.metadata.version('werkit')}",
+            f"werkit=={__version__}",
```

Comment:
I'd suggest we remove this `install_werkit` feature. It's a breaking change but it's a fiddly feature to maintain – keeping the version synced in package_version doesn't feel worth the effort. And it's of questionable value to begin with, since most lambdas will require their own dependency management (using poetry or requirements files), and those dependency management systems can specify werkit as a dependency, too.

Reply:
Fixed in ce4698eb0c1b174f863df318c82f63d4d6c46f43 and https://github.com/metabolize/purl-anatomia/pull/446/commits/8fb9faf1dc7728c47d8d88173952212eaccde80b